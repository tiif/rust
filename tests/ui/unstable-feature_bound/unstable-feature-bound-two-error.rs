//@ aux-build:unstable_feature.rs
extern crate unstable_feature;
use unstable_feature::{Foo, Bar, Moo};

// FIXME: both `feat_bar` and `feat_moo` are needed to pass this test,
// but the diagnostic only will point out `feat_bar`.

fn main() {
    Bar::foo();
    //~^ ERROR: unstable feature `feat_bar` is used without being enabled.
    Moo::foo();
}
