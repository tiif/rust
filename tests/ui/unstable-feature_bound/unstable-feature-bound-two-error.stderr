error: unstable feature `feat_bar` is used without being enabled.
  --> $DIR/unstable-feature-bound-two-error.rs:9:5
   |
LL |     Bar::foo();
   |     ^^^
   |
   = help: The feature can be enabled through #[unstable_feature_bound(feat_bar)] in std/core, or with #[feature(feat_bar)] outside of std/core.
   = note: required for `Bar` to implement `Foo`

error: aborting due to 1 previous error

